src/api/routes.ts

```diff
@@ -76,10 +83,10 @@ export const buildDeleteItemsRoute = (ids: UUID[]) =>
       addQueryPrefix: true,
     },
   )}`;
-export const buildGetChildrenRoute = (id: UUID, ordered: boolean) =>
+export const buildGetChildrenRoute = (id: UUID, params: ItemChildrenParams) =>
   `${ITEMS_ROUTE}/${id}/children${qs.stringify(
-    { ordered },
-    { addQueryPrefix: true },
+    { ...params },
```

Comment:
Do you need to spread because it can be undefined ? I think at that stage it should not be undefined anymore, can you try: 
```suggestion
    params,
```

Reply:
No it is because I forget to remove their when I added ordered in params